src/js/components/Navigation/MainHeaderBar.jsx

```diff
@@ -98,7 +98,7 @@ export default function MainHeaderBar () {
             <Suspense fallback={<span>ZzZzZzZ</span>}>
               <SignInButton classes={classes} />
```

Comment:
I'm assuming that this button is only visible directly on the main header bar for testing purposes. When you are done with testing, it should be moved into the drop-down so we would show either "Sign In" or "Sign Out" at the bottom of the drop-down.